css/properties.json

```diff
@@ -703,24 +707,24 @@
     "groups": [
       "WebKit Extensions"
     ],
-    "initial": "<code>rgb(0, 0, 0)</code>",
+    "initial": "<code>currentColor</code>",
     "appliesto": "allElements",
-    "computed": "asSpecified",
+    "computed": "'color'",
     "order": "uniqueOrder",
     "status": "nonstandard"
   },
   "-webkit-text-stroke-width": {
     "syntax": "&lt;length&gt;",
     "media": "visual",
     "inherited": true,
-    "animationType": "discrete",
+    "animationType": "no",
```

Comment:
Please keep `"discrete"`. The W3C [decided some time ago](https://lists.w3.org/Archives/Public/www-style/2015Nov/0308.html) to replace the "animatable" info by an "animation type" [with "discrete" as value](https://lists.w3.org/Archives/Public/www-style/2016May/0121.html) when they have a discrete value transition.

The spec. obviously didn't get updated in this regard yet.

Sebastian